submissions/lisnyak-o/popup-kottans/index.html

```diff
@@ -19,10 +19,10 @@
   <header class="header">
     <div class="container">
       <nav class="menu">
-        <ul class="menu__list menu__wrapper">
+        <ul class="menu__list menu--wrapper">
```

Comment:
[Read more about BEM modificator](https://ru.bem.info/methodology/quick-start/#%D0%9C%D0%B8%D0%BA%D1%81)
If you want to use this naming you should use something like this: `menu__list menu__list--modificator`

Reply:
Sure, I change mistakes, thanks you)